app/controllers/consent_forms_controller.rb

```diff
@@ -56,6 +56,7 @@ def create_patient
 
     ActiveRecord::Base.transaction do
       @patient.save!
+      @patient.sessions << @consent_form.original_session
```

Comment:
Just thinking about this more, I think we'll need to re-create this logic here: https://github.com/nhsuk/manage-vaccinations-in-schools/blob/ff12ec76d61b7b37b163c371a3c7555cb364dd04/app/models/consent_form.rb#L431-L439

But I'm also wondering if it makes more sense to make `actual_upcoming_session` return the community clinic rather than `nil`, then in both places we don't need to re-implement the logic.

Reply:
Done!